src/main/kotlin/com/atlassian/performance/tools/report/jstat/JstatConverter.kt

```diff
@@ -19,7 +19,7 @@ internal class JstatConverter {
                 it.lines()
                     .skip(1)
                     .map { it.replace(Regex(" +"), ",") }
-                    .forEach { jstatCsv.toFile().appendText(it + "\n") }
+                    .forEach { jstatCsv.toFile().appendText(it.replace("-,","0,") + "\n") }
```

Comment:
We can simplify this quite busy line by moving the transformation into its own stream operation:
```suggestion
                    .map { it.replace("-," ,"0,") }
                    .forEach { jstatCsv.toFile().appendText("$it\n") }
```